modules/dnn/test/test_common.hpp

```diff
@@ -237,49 +213,31 @@ namespace opencv_test {
 using namespace cv::dnn;
 
 static inline
-testing::internal::ParamGenerator<tuple<Backend, Target> > dnnBackendsAndTargets(
+testing::internal::ParamGenerator< tuple<Backend, Target> > dnnBackendsAndTargets(
         bool withInferenceEngine = true,
         bool withHalide = false,
         bool withCpuOCV = true
```

Comment:
`withCpuOCV` is ignored here. Some tests use default CPU implementation as reference so we don't compare it with itself.